modules/i3s/test/i3s-attributes-loader.spec.js

```diff
@@ -0,0 +1,92 @@
+import test from 'tape-promise/tape';
+import I3SAttributesLoader from '../src/i3s-attributes-loader';
+
+const attributeStorageInfo = [
```

Comment:
+100, love the tests, thank you!!!

Reply:
> This looks good. I assume some of this attribute loading is already happening in the I3SLoader (after all we are able to render I3S tiles today).
> 
> So I assume the next PR will replace existing code with this loader?
> 
> It could be useful if you explain such context / plans in the PR description.

There are so many `attributes` words related to `I3SLoader`.
Already existing attribute loading functionality in `I3SLoader` is responsible for loading `geometry attributes` such as:
![image](https://user-images.githubusercontent.com/70207219/95728551-4e12bf80-0c84-11eb-95fb-3c893e1fdf35.png)

My new loader is responsible for `feature attributes` which is not the same as previous.
This attributes characterize particular feature (building or some object on the map).
For example:
![image](https://user-images.githubusercontent.com/70207219/95728932-cd07f800-0c84-11eb-9ca3-5aa7610525b2.png)